packages/consoledot-api/src/queries/UseUser.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from "@tanstack/react-query";
+import { masQueries } from "../queryKeys";
+import { useChrome } from "@redhat-cloud-services/frontend-components/useChrome";
+
+export const useUser = () => {
+  const { auth } = useChrome();
+  const { data } = useQuery<{ username: string; isOrgAdmin: boolean }>({
+    queryKey: masQueries.userName(),
+    queryFn: async () => {
+      const user = await auth.getUser();
+      return {
+        username: user?.identity?.user?.username ?? "",
+        isOrgAdmin: user?.identity?.user?.is_org_admin ?? false,
+      };
+    },
+  });
+
+  return data ?? { username: "", isOrgAdmin: false };
```

Comment:
I'm not a fan of this. It's better to return the returned value of `useQuery` directly, so we have access to all the other properties that the hook gives us (all the `isXXX` booleans for example).

As I commented elsewhere to Suyash, the fact that data might be unavailable is a problem we have to deal with, we can't just default values and call it a day.